.pre-commit-config.yaml

```diff
@@ -111,6 +111,12 @@ repos:
     hooks:
       - id: black
 
+  - repo: https://github.com/tox-dev/pyproject-fmt
+    rev: "1.0.0"
+    hooks:
+      - id: pyproject-fmt
+        args: ["--indent=4"]
```

Comment:
Unfortunately, I cannot figure out how to independently configure the indent setting (requested by @pllim) outside of specifying it via command line args. I have opened tox-dev/pyproject-fmt#130 for assistance with this.